### PR TITLE
chore: create empty ingest package

### PR DESCRIPTION
**Pull request by mantagen**
## Description

- Creates an empty package for the ingest service
- Adds graphql query for Oak lessons
- Adds functions (largely copied from OWA) for fetching captions

### CODE REVIEW

File: packages/ingest/src/import-lessons/graphql/client.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { GraphQLClient } from "graphql-request";
+
+if (!process.env.OAK_GRAPHQL_SECRET) {
+  throw new Error("OAK_GRAPHQL_SECRET is not set");
+}
+
+export const graphqlClient = new GraphQLClient(
+  `https://hasura.thenational.academy/v1/graphql`,
```
Comment (mantagen) [inline packages/ingest/src/import-lessons/graphql/client.ts:8]: This is already in the repo in packages/core.

File: packages/ingest/src/captions/getCaptionsByFileName.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  const sentence1 = sentences[0]!.replace("<v ->", "").replace("</v>", "");
+  return [sentence1, ...sentences.slice(1)];
+};
+
+const getTextFromCues = (cueList: Cue[]): Array<string> => {
+  return Object.values(cueList).map((cue) => cue.text);
+};
+
+async function getFileFromBucket(bucketName: string, fileName: string) {
+  const client = new Storage();
```
Comment (mantagen) [inline packages/ingest/src/captions/getCaptionsByFileName.ts:65]: I think locally I've been using silent auth. Haven't set up credentials yet for this to be able run in the cloud

File: packages/ingest/src/import-lessons/graphql/client.ts (hunk 1 above)
Comment (codeincontext) [inline packages/ingest/src/import-lessons/graphql/client.ts:8]: At some point it would be good to put this behind an env var

**Review by codeincontext — APPROVED** [review]
Looking good so far. I appreciate the way you're dripping in PRs for this

File: packages/ingest/src/import-lessons/graphql/client.ts (hunk 1 above)
Comment (mantagen) [reply packages/ingest/src/import-lessons/graphql/client.ts:8]: Agreed

**Pull request merged**